docs/content-services/components/search-date-range-advanced.component.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Search date range advanced component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-13
+---
+
+# [Search date range advanced component](../../../lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.ts "Defined in search-date-range.component.ts")
```

Comment:
This path is for search-date-range-advanced-tabbed.component.ts but documentation is for search-date-range-advanced.component

Reply:
Updated documentation. Probably copy paste error. Thanks!